src/io/terminal.cpp

```diff
@@ -76,3 +81,10 @@ void terminal_write(const char* data) {
 		terminal_putchar(data[i]);
 }
 
+void terminal_println(const char* data){
+	terminal_write(data);
+	current_terminal_column = 0;
+	if( ++current_terminal_row == VGA_HEIGHT)
```

Comment:
The code that checks whether the row has gone past the height of the view, which is shared across these two functions may be nice to consolidate into its own function, something like "terminal_next_line" This will also be nice since then we can handle going above the height in this one location in the future when we may decide to implement something like scrolling